### PR TITLE
Page template

### PR DESCRIPTION
**Pull request by annatraussnig**
Screenshot 2020-10-07 at 15 52 18

- new `page` schema
- corresponding docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,7 +126,8 @@
   },
   "dependencies": {
     "body-scroll-lock": "3.0.3",
-    "react-icons": "^2.2.7"
+    "react-icons": "^2.2.7",
+    "typescript": "^4.0.3"
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply package.json]: not sure... i think i meant to install it globally... anyway...

File: src/templates/Article/index.js [hunk 2]
```diff
@@ -160,9 +160,10 @@ const createSchema = ({
   previewTeaser,
   getVideoPlayerProps = props => props,
   onAudioCoverClick,
-  metaBody = false
+  metaBody = false,
+  useInteractionsTypo = false
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply src/templates/Article/index.js]: thanks for suggestion - makes a lot of sense.

File: src/templates/Page/docs.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- `skipCredits`: true
+- `excludeFromFeed`: true
+- `titleCenter`: true (initial default - can always be changed)
+
+And offers the following custom meta configuration options:
+- `disableActionBar`: default false
+- `discussion`: (optional) discussion repo (if set, will show up in action bar) 
+
+Defaults:
+- `repoPrefix`, `static-`
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply src/templates/Page/docs.md]: good catch, thanks!

File: package-lock.json [hunk 4]
```diff
@@ -22870,6 +22870,11 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=",
       "dev": true
     },
+    "typescript": {
```
Comment (tpreusse) [inline package-lock.json:22873]: `package-lock.json` still has typescript and https to http changes, should be reset to master too.

**Review by tpreusse — APPROVED** [review]

File: package-lock.json (hunk 4 above)
Comment (annatraussnig) [reply package-lock.json]: my bad

**Pull request merged**